modules/alloy/src/main/ts/ephox/alloy/positioning/view/Callouts.ts

```diff
@@ -54,7 +56,7 @@ const setWidth = (element: SugarElement, decision: RepositionDecision, options:
 const position = (element: SugarElement, decision: RepositionDecision, options: ReparteeOptions): void => {
   // This is a point of difference between Alloy and Repartee. Repartee appears to use Measure to calculate the available space for fixed origin
   // That is not ported yet.
-  applyPositionCss(element, Origins.reposition(options.origin, decision));
+  applyPositionCss(element, Origins.reposition(options.origin, decision), Optional.some(decision));
```

Comment:
I'm trying to understand why `decision` is being wrapped in an `Optional` now?

It appears that it is not touched until it reaches `PositionUtils.ts` where it is used like this:


```typescript
const isDecisionTopAligned = (decisionOpt: Optional<RepositionDecision>): boolean => {
  return decisionOpt.map((decision) => decision.label === north).getOr(false);
};
```

But couldn't we remove the optional and do this:

```typescript
const isDecisionTopAligned = (decision: RepositionDecision): boolean => {
  return decision.label === north;
};
```

Am I missing something?

Reply:
No, in `DockingApis.ts` we have the code
```
const morphToCoord = (component: AlloyComponent, config: DockingConfig, position: PositionCss): void => {
  applyPositionCss(component.element, position, Optional.none());
  const method = position.position === 'fixed' ? config.onDocked : config.onUndocked;
  method(component);
};
```
Which does not have a decision, so `Optional.none()` is used.